packages/devtools_app/lib/src/screens/memory/memory_protocol.dart

```diff
@@ -23,51 +23,51 @@ import 'memory_timeline.dart';
 class MemoryTracker {
   MemoryTracker(this.serviceManager, this.memoryController);
 
-  ServiceConnectionManager serviceManager;
+  ServiceConnectionManager? serviceManager;
```

Comment:
remove this var. We should be using the serviceManager global, not passing it through method calls and constructors